test/accessibility/services/a11y/constants.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ReporterVersion } from 'axe-core';
+
+export const AXE_CONFIG = {
+  rules: [
+    {
+      id: 'scrollable-region-focusable',
+      selector: '[data-skip-axe="scrollable-region-focusable"]',
+    },
+    {
+      id: 'aria-required-children',
+      selector: '[data-skip-axe="aria-required-children"] > *',
+    },
+    {
+      id: 'label',
+      selector: '[data-test-subj="comboBoxSearchInput"] *',
+    },
+    {
+      id: 'aria-roles',
+      selector: '[data-test-subj="comboBoxSearchInput"] *',
+    },
+    {
+      // EUI bug: https://github.com/elastic/eui/issues/4474
+      id: 'aria-required-parent',
+      selector: '[class=*"euiDataGridRowCell"][role="gridcell"]',
+    },
+    {
+      // 3rd-party library; button has aria-describedby
+      id: 'button-name',
+      selector: '[data-rbd-drag-handle-draggable-id]',
+    },
+    {
+      // EUI bug: https://github.com/elastic/eui/issues/4536
+      id: 'duplicate-id',
+      selector: '.euiSuperDatePicker *',
+    },
+  ],
+};
+
+export const AXE_OPTIONS = {
+  reporter: 'v2' as ReporterVersion,
```

Comment:
FYI without this `as`, Typescript was throwing errors that 'v2' was parsing as a string and not ReporterVersion, so I had to cast it here. Good time with string literals

Reply:
Not sure if you want to take this on in this PR but we wrote our own TS definitions for axe and that's what we still use but since then axe has released their own definitions which are probably more accurate 😅 Not sure how big of an issue the upgrade might be but it might fix this?

But nbd either way

Reply:
Ahh super good to know/have that context! Unfortunately I'll probably shy away from an upgrade in this PR due to impending timelines 😅 Sorry!!